exonum/src/events/internal.rs

```diff
@@ -71,6 +76,23 @@ impl InternalPart {
                         }).map_err(|_| panic!("Can't execute shutdown"));
                         to_box(f)
                     }
+                    InternalRequest::VerifyTx(tx) => {
+                        let internal_tx = internal_tx.clone();
```

Comment:
Cloning an mpsc sender to send a single message looks inefficient (although it may be not, profiling could be helpful). I wonder if it's possible and makes sense to use `thread_local` crate here (or upgrade to `futures >0.1`).